portfolio/src/main/webapp/style.css

```diff
@@ -4,6 +4,48 @@
   width: 650px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+* {
+    box-sizing: border-box;
 }
+
+body {
+    font-family: Verdana, sans-serif;
+}
+
+/* Slideshow container */
+.slideshow-container {
+  max-width: 1000px;
+  position: relative;
+  margin: auto;
+  align-items: center;
+}
+
+/* Hide slides at first */
+.slide {
+    display: none;
+}
+
+.prev, .next {
```

Comment:
Separate selectors and declarations by new lines.
https://google.github.io/styleguide/htmlcssguide.html#Selector_and_Declaration_Separation

Reply:
done :)